Cookbook.Domain/Models/Ingredient.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Cookbook.Domain.Models
+{
+    public class Ingredient
+    {
+        public required int Amount { get; init; }
```

Comment:
Ezt beszeltuk mar, de en nem igy modelleztem volna. Amount-nak a recipe-be kene mennie, nem akarnek 10 ingredient-et a Milk-hez, csak mert az egyik 1 deci a masik 2. Meg mi van ha van 1000 ml-es milk meg 1 literes milk?